packages/publish-views/pipeline/steps/loadViews.ttl

```diff
@@ -0,0 +1,11 @@
+PREFIX code: <https://code.described.at/>
+prefix : <https://pipeline.described.at/>
+
+<#loadViews>
+  a :Step ;
+  code:implementedBy
+    [
+      code:link <file:../lib/loadViews.js#default> ;
+      a code:EcmaScriptModule ;
+    ] ;
+.
```

Comment:
Moved the implementation of this step to separate file so that a different one can be substituted in test